apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.html

```diff
@@ -18,19 +18,22 @@
                             <table>
                                 <tbody>
                                     <tr>
-                                        <td class="text-right width-70" ng-class="{width55: ledgerCtrl.selectedTxn.panel.unit != undefined}">
+                                        <td class="text-right width-70"
+                                            ng-class="{width55: ledgerCtrl.selectedTxn.panel.unit != undefined}">
```

Comment:
replace ng-class by [ngClass] and put single quote around the class name width55